<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PL_mainDashboard</class>
 <widget class="QMainWindow" name="PL_mainDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #414752;</string>
  </property>
  <widget class="QWidget" name="wdg_main_admin">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frm_menu_bar">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#313134;
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFrame" name="frm_logo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_logo_pipeliam">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color:transparent;border:none}

QPushButton:hovered{background-color:transparent;border:none}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../utils/utils.qrc">
              <normaloff>:/logo/logo/logo_green.png</normaloff>:/logo/logo/logo_green.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frm_menu_bar_main">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>332</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_logout">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color:transparent;
	color:#414752;
}
QPushButton:hover{
	color:#313134;
	icon: url(:/icons/icons/logout_h.png);
	background-color:transparent;

}
QPushButton:pressed{
	background-color:transparent;
	color:#FFF;	
    icon: url(:/icons/icons/logout_h.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../utils/utils.qrc">
              <normaloff>:/icons/icons/logout.png</normaloff>:/icons/icons/logout.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frm_menu">
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="vs_menu_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frm_menu_back">
         <property name="styleSheet">
          <string notr="true">background-color : #626A79;
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frm_main">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frm_top">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lab_welcome">
            <property name="font">
             <font>
              <family>TrendSansW00-One</family>
              <pointsize>35</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color :#37ad74</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>WELCOME &quot;USER&quot;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frm_bot">
         <property name="styleSheet">
          <string notr="true">QFrame{background-color:#626A79;border-radius:10px}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frm_project">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color:#8B94A4;
	border-style: solid;
	border-color: transparent;
	border-width: 5px;
	border-radius: 17px;
	margin-left : 60px;
	margin-right : 60px;
	color:#414752;
}
QPushButton:hover{
	color:#313134;
	background-color:#FFF;

}
QPushButton:pressed{
	background-color:#313134;
	color:#FFF;	
    icon: url(:/icons/icons/folder_h.png);
	margin-left : 65px;
	margin-right : 65px;
	margin-top: 5px;
	margin-bottom : 5px;
}

QLabel:hover{
	color : #FFF;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_project">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>250</height>
                </size>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:#8B94A4;
	border-style: solid;
	border-color: transparent;
	border-width: 5px;
	border-radius: 17px;
	margin-left : 60px;
	margin-right : 60px;
	color:#414752;
}
QPushButton:hover{
	color:#313134;
	background-color:#FFF;

}
QPushButton:pressed{
	background-color:#313134;
	color:#FFF;	
    icon: url(:/icons/icons/folder_h.png);
	margin-left : 65px;
	margin-right : 65px;
	margin-top: 5px;
	margin-bottom : 5px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../utils/utils.qrc">
                 <normaloff>:/icons/icons/folder.png</normaloff>:/icons/icons/folder.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lab_project">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>TrendSansW00-One</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color :#414752;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Projects</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignHCenter|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frm_library">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_library">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>250</height>
                </size>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:#8B94A4;
	border-style: solid;
	border-color: transparent;
	border-width: 5px;
	border-radius: 17px;
	margin-left : 60px;
	margin-right : 60px;
	color:#414752;
}
QPushButton:hover{
	color:#313134;
	background-color:#FFF;

}
QPushButton:pressed{
	background-color:#313134;
	color:#FFF;	
    icon: url(:/icons/icons/images_h.png);
	margin-left : 65px;
	margin-right : 65px;
	margin-top: 5px;
	margin-bottom : 5px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../utils/utils.qrc">
                 <normaloff>:/icons/icons/images.png</normaloff>:/icons/icons/images.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lab_library">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>TrendSansW00-One</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color :#414752;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Library</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignHCenter|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frm_team">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_team">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>250</height>
                </size>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:#8B94A4;
	border-style: solid;
	border-color: transparent;
	border-width: 5px;
	border-radius: 17px;
	margin-left : 60px;
	margin-right : 60px;
	color:#414752;
}
QPushButton:hover{
	color:#313134;
	background-color:#FFF;

}
QPushButton:pressed{
	background-color:#313134;
	color:#FFF;	
    icon: url(:/icons/icons/user_h.png);
	margin-left : 65px;
	margin-right : 65px;
	margin-top: 5px;
	margin-bottom : 5px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../utils/utils.qrc">
                 <normaloff>:/icons/icons/user.png</normaloff>:/icons/icons/user.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lab_team">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>TrendSansW00-One</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color :#414752;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>Team</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignHCenter|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>TrendSansW00-One</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #FFF;background-color: #313134;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../utils/utils.qrc"/>
 </resources>
 <connections/>
</ui>
